src/ape_console/_cli.py

```diff
@@ -24,6 +26,24 @@ def cli(cli_ctx, network):
     return console(verbose=verbose)
 
 
+def load_consolerc(namespace):
+    """load and return namespace from consolerc.py if it exists"""
+    consolerc = config.DATA_FOLDER.joinpath("consolerc.py")
```

Comment:
Instead of the ape data folder, would it make sense for the path to be config-file driven?
The reason is because I don't think users typically add stuff to the application data folders

Reply:
Do you have an example somewhat populated config file I can check out?

Reply:
Yes, there are lots of example keys in this guide: https://docs.apeworx.io/ape/latest/userguides/config.html

Here is a real example: https://github.com/unparalleled-js/ape-ethdenver/blob/main/project/ape-config.yaml